### PR TITLE
about.html: Add content in cards

### PR DESCRIPTION
**Pull request by hemangsk**
screencapture-localhost-8080-1493371121558

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [ ] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [ ] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

File: partials/tabs/about.html [hunk 1]
```diff
@@ -22,8 +22,8 @@
                languages supported and the bears themselves, click here.</p>
           </div>
           <div class="card-action">
-            <a href="#">Bears Documentation</a>
-            <a href="#">Bears Source</a>
+            <a href="https://docs.coala.io/en/latest/">Documentation</a>
+            <a href="https://github.com/coala/coala/">Source</a>
```
Comment (sims1253) [inline partials/tabs/about.html]: shouldn't this be https://github.com/coala/coala-bears?

File: partials/tabs/about.html [hunk 2]
```diff
@@ -22,8 +22,8 @@
                languages supported and the bears themselves, click here.</p>
           </div>
           <div class="card-action">
-            <a href="#">Bears Documentation</a>
-            <a href="#">Bears Source</a>
+            <a href="https://docs.coala.io/en/latest/">Documentation</a>
```
Comment (sims1253) [inline partials/tabs/about.html]: This shall still point to the bears doc right? Is the general docs the right place to point at?

File: partials/tabs/about.html (hunk 1 above)
Comment (hemangsk) [inline partials/tabs/about.html]: oops, fixing it!

**Comment by sils** [comment]
ack 2f0e8ac8797bdb8d8d833e97340c9c9a444a7c2d
ack f4bf3ca2bf66c863ac1bada0826b28a00ad318cc
ack 6a48e21ce255b8b841d96a06a2276cd9b0cd2511

**Pull request merged**